mailbox/elasticsearch-v7/src/test/java/org/apache/james/mailbox/elasticsearch/v7/json/MessageToElasticSearchJsonTest.java

```diff
@@ -132,6 +133,7 @@ void invalidCharsetShouldBeWellConvertedToJson() throws IOException {
     }
 
     @Test
+    @Disabled
```

Comment:
I'm curious to know why these tests are failing as they do not directly depends on ElasticSearch but as far as I know on Jackson...

Reply:
They failed previously, something wrong with my environment. I removed the @Disabled and the tests passed.

Reply:
https://github.com/apache/james-project/pull/288/commits/16c4df23588581471bc96b3cc6be732e6d12cfde